src/Template/Bake/Template/index.ctp

```diff
@@ -39,20 +39,21 @@ $fields = collection($fields)
     endforeach;
 %>
     </ul>
-</div>
-<div class="<%= $pluralVar %> index large-10 medium-9 columns">
+</nav>
+<div class="<%= $pluralVar %> index large-9 medium-8 columns" id="content">
+    <h3><?= __('<%= $pluralHumanName %>') ?></h3>
     <table cellpadding="0" cellspacing="0">
```

Comment:
Cellspacing and cellpadding can be removed. See http://stackoverflow.com/a/3209434